### PR TITLE
Avro sample

### PR DESCRIPTION
**Pull request by gurbuzali**
depends on https://github.com/hazelcast/hazelcast-jet/pull/890

### CODE REVIEW

File: batch/avro-file/src/main/java/SinkSample.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+
+/**
+ * Demonstrates dumping a map's values to an avro file
```
Comment (viliam-durina) [inline batch/avro-file/src/main/java/SinkSample.java]: We should use capitalized "Avro" and a period at the end. Also in the SourceSample.

File: batch/avro-file/src/main/java/SourceSample.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.Util;
+import com.hazelcast.jet.avro.AvroSources;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import model.User;
+
+/**
+ * Demonstrates reading avro files from a directory and populating IMap
+ * Run {@link SinkSample} first to create necessary avro files directory
```
Comment (viliam-durina) [inline batch/avro-file/src/main/java/SourceSample.java]: This should be also in code. If the directory doesn't exist, ask the user to run SinkSample first and exit.

File: batch/avro-file/src/main/java/SourceSample.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ */
+public class SourceSample {
```
Comment (viliam-durina) [inline batch/avro-file/src/main/java/SourceSample.java]: I'd name these classes `AvroSourceSample` and `AvroSinkSample`.

**Comment by gurbuzali** [comment]
verify

**Pull request merged**